docker/qr/http/index.html

```diff
@@ -106,7 +106,7 @@
             version = '';
         }
         let ruffleScript = document.createElement("script");
-        ruffleScript.setAttribute("src", `https://unpkg.com/@ruffle-rs/ruffle${version}`);
```

Comment:
I believe the interpolation won't work without the `$` symbol

Reply:
Whoops, done